release-notes/unreleased/main.yml

```
@@ -0,0 +1,6 @@
+issue_key:
+show_in_stores: true
+platforms:
+  - android
+  - ios
+de: Wiederholungsmechanismus bei fehlerhaften Eingaben für alle Aufgabentypen
```

`Wiederholungsmechanismus` is quite a difficult word, maybe:
```suggestion
de: Falsche Eingaben werden nun bei allen Übungen wiederholt
```